pkg/app/piped/platformprovider/lambda/client.go

```diff
@@ -278,6 +281,9 @@ func (c *client) updateFunctionConfiguration(ctx context.Context, fm FunctionMan
 		if fm.Spec.Handler != "" {
 			configInput.Handler = aws.String(fm.Spec.Handler)
 		}
+		if fm.Spec.EphemeralStorage.Size != 0 {
```

Comment:
```suggestion
		if fm.Spec.EphemeralStorage != nil && fm.Spec.EphemeralStorage.Size != 0 {
```
avoid nil pointer error due to the omitempty json tag

Reply:
Thank you! I added it.
[add nil pointer check](https://github.com/pipe-cd/pipecd/pull/4680/commits/caf132172af73103066822359604429ef3840f74)